README.md

```diff
@@ -10,7 +10,170 @@ framework, including Polkadot parachains.
 
 ## How to integrate the Filecoin bridge pallet into a runtime?
 
-_TODO_
+### 0. add `filecoindot` to your runtime config
+
+```
+// Cargo.toml
+
+filecoindot = { git = "https://github.com/chainSafe/filecoindot",  default-features = false }
+```
+
+Here you need to register filecoindot and the offchain worker's logic to your `runtime.rs`:
+
+```rust
+// runtime.rs
+
+parameter_types! {
+    pub const OffchainWorkerTimeout: u64 = 1_000_000;
+}
+
+// ManagerOrigin as root
+type ManagerOrigin = frame_system::EnsureRoot<AccountId>;
+
+impl filecoindot::Config for Runtime {
+    type ManagerOrigin = ManagerOrigin;
+    type Event = Event;
+    type WeightInfo = ();
+    type AuthorityId = filecoindot::FilecoindotId;
+    type OffchainWorkerTimeout = OffchainWorkerTimeout;
+}
+
+// For pallet-example-offchain-worker
+parameter_types! {
+    pub const GracePeriod: BlockNumber = 3;
+    pub const UnsignedInterval: BlockNumber = 3;
+    pub const UnsignedPriority: BlockNumber = 3;
+}
+
+impl<LocalCall> frame_system::offchain::CreateSignedTransaction<LocalCall> for Runtime
+where
+    Call: From<LocalCall>,
+{
+    fn create_transaction<C: frame_system::offchain::AppCrypto<Self::Public, Self::Signature>>(
+        call: Call,
+        public: <Signature as sp_runtime::traits::Verify>::Signer,
+        account: AccountId,
+        index: Index,
+    ) -> Option<(
+        Call,
+        <UncheckedExtrinsic as sp_runtime::traits::Extrinsic>::SignaturePayload,
+    )> {
+        let period = BlockHashCount::get() as u64;
+        let current_block = System::block_number()
+            .saturated_into::<u64>()
+            .saturating_sub(1);
+        let tip = 0;
+        let extra: SignedExtra = (
+            frame_system::CheckSpecVersion::<Runtime>::new(),
+            frame_system::CheckTxVersion::<Runtime>::new(),
+            frame_system::CheckGenesis::<Runtime>::new(),
+            frame_system::CheckEra::<Runtime>::from(generic::Era::mortal(period, current_block)),
+            frame_system::CheckNonce::<Runtime>::from(index),
+            frame_system::CheckWeight::<Runtime>::new(),
+            pallet_transaction_payment::ChargeTransactionPayment::<Runtime>::from(tip),
+        );
```

Comment:
this looks pretty verbose,

can we either,

* exclude this from the overview?
* explain whats happening/why we need this?
* link to existing offchain worker docs?